src/styles.css

```diff
@@ -0,0 +1,159 @@
+@import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro&display=swap");
+.App {
+  font-family: sans-serif;
+}
+
+.container{
+  margin: auto;
+  width: 600px;
+  height: 400px;
+  position: relative;
+  text-align: center;
+  padding: 15px;
+  box-shadow: 0 2px 30px rgba(255, 255, 255, 0);
+  color: rgb(219, 139, 163);
+  
+}
+
+.clock {
+  width: 300px;
+  height: 300px;
+  border-radius: 50%;
+  position: absolute;
+  top: 20%;
+  left: 25%;
+  /* transform: translate(-50%, -50%); */
+  box-shadow: 0 2px 30px rgba(0, 0, 0, 0.2);
+  font-size: 24px;
+  color: #444;
+  text-align: center;
+}
+
+.clock::after {
+  background: rgb(86, 142, 247);
+  content: "";
+  width: 12px;
+  height: 12px;
+  border-radius: 50%;
+  position: absolute;
+  z-index: 2;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  border: 2px solid #fff;
+}
+
+.hour_hand {
+  position: absolute;
+  width: 6px;
+  height: 60px;
+  background: rgb(86, 142, 247);
+  top: 30%;
+  left: 49%;
+  transform-origin: bottom;
+}
+
+.min_hand {
+  position: absolute;
+  width: 4px;
+  height: 80px;
+  background: rgb(86, 142, 247);
+  top: 22.5%;
+  left: 49%;
+  transform-origin: bottom;
+}
+
+.sec_hand {
+  position: absolute;
+  width: 2px;
+  height: 118px;
+  background: rgb(86, 142, 247);
+  top: 10.5%;
+  left: 50%;
+  transform-origin: bottom;
+}
+
+.clock span {
+  position: absolute;
+  font-family: "Source Sans Pro", sans-serif;
+  font-weight: 700;
+}
+
+.twelve {
+  top: 10px;
+  left: 46%;
+  color: rgb(86, 142, 247);
+}
+
+.one {
+  top: 10%;
+  right: 26%;
+  color: rgb(86, 142, 247);
+}
+
+.eleven {
+  top: 10%;
+  left: 26%;
+  color: rgb(86, 142, 247);
+}
+
+.two {
+  top: 25%;
+  right: 10%;
+  color: rgb(86, 142, 247);
+}
+
+.three {
+  right: 10px;
+  top: 46%;
+  color: rgb(86, 142, 247);
+}
+
+.four {
+  right: 30px;
+  top: 67%;
+  color: rgb(86, 142, 247);
+}
+
+.five {
+  right: 78px;
+  top: 80%;
+  color: rgb(86, 142, 247);
+}
+
+.six {
+  bottom: 10px;
+  left: 50%;
+  color: rgb(86, 142, 247);
+}
+
+.seven {
+  left: 80px;
+  top: 82%;
+  color: rgb(86, 142, 247);
+}
+
+.eight {
+  left: 30px;
+  top: 67%;
+  color: rgb(86, 142, 247);
+}
+
+.nine {
+  left: 10px;
+  top: 46%;
+  color: rgb(86, 142, 247);
+}
+
+.ten {
+  top: 25%;
+  left: 10%;
+  color: rgb(86, 142, 247);
+}
```

Comment:
I think at this point you should research [CSS Variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) for the color, and consider either using SCSS or generating these styles from JS.
Also, when mixing `%` and `px` in absolute layouts, you're doomed to terrible responsiveness. Using `%` would most likely be the better choice.

Reply:
:root {
  --numbers-color: rgb(86, 142, 247);
  --hands-color: rgb(86, 142, 247);
  --timeSetTo-color:rgb(219, 139, 163);
  --number-font-size: 22px;
}

.button12 {
...
  top: 5%;
  left: 45%;